16_microbiome_community.Rmd

```diff
@@ -49,24 +49,220 @@ Community composition can be visualized with heatmap where other axis represent
 samples and another taxa. Color of each intersection point represent abundance
 of a taxon in a specific sample. 
 
-Here, Z-transformed abundances are plotted at Phylum level. 
+Here, abundances are first CLR (centered log ratio) transformed, and then 
+Z transformation is applied to CLR-transformed data. After that abundances are 
+plotted at Phylum level. 
 
 ```{r heatmap}
 library(ggplot2)
+# Does clr-transformation
+tse_phylum <- transformSamples(tse_phylum, method = "clr", pseudocount = 1)
 # Does z-transformation
-tse_phylum <- transformFeatures(tse_phylum, method = "z", name = "Ztransform")
+tse_phylum <- transformFeatures(tse_phylum, method = "z", name = "clr_z")
```

Comment:
Should in this case  'abund_values="clr" ' be mentioned?
When I checked the documentation it uses "counts" as default.

Reply:
Yes, good. Thanks!

Reply:
Fixed